authenticators/cf_authenticator_test.go

```diff
@@ -92,6 +93,49 @@ var _ = Describe("CFAuthenticator", func() {
 		})
 	})
 
+	Describe("Authenticate invalid token returned", func() {
+		const expectedOneTimeCode = "abc123"
+
+		var (
+			uaaTokenResponse     *authenticators.UAAAuthTokenResponse
+			uaaTokenResponseCode int
+		)
+
+		BeforeEach(func() {
+			metadata.UserReturns("cf:1e051b88-a210-40b7-bcca-df645b24b634/1")
+			password = []byte(expectedOneTimeCode)
+
+			uaaTokenResponseCode = http.StatusOK
+			uaaTokenResponse = &authenticators.UAAAuthTokenResponse{
+				AccessToken: "is not right",
+				TokenType:   "bearer",
+			}
+
+			fakeUAA.AppendHandlers(
+				ghttp.CombineHandlers(
+					ghttp.VerifyRequest("POST", "/oauth/token"),
+					ghttp.VerifyBasicAuth("diego-ssh", "fake-diego-ssh-secret-$\"^&'"),
+					ghttp.VerifyFormKV("grant_type", "authorization_code"),
+					ghttp.VerifyFormKV("code", expectedOneTimeCode),
+					ghttp.RespondWithJSONEncodedPtr(&uaaTokenResponseCode, uaaTokenResponse),
+				),
+			)
+		})
+
+		It("logs the access to the container by the user", func() {
+			Expect(authenErr).To(Equal(authenticators.AuthenticationFailedErr))
+			Expect(fakeCC.ReceivedRequests()).To(HaveLen(0))
+			// Eventually(logger).Should(gbytes.Say("test.cf-authenticate.app-access-success"))
+			// for _, msg := range logger.Logs() {
+			// 	if msg.Message == "test.cf-authenticate.app-access-success" {
+			// 		Expect(msg.Data["app"]).To(Equal("1e051b88-a210-40b7-bcca-df645b24b634/1"))
+			// 		Expect(msg.Data["username"]).To(Equal("admin"))
+			// 		Expect(msg.Data["principal"]).To(Equal("36ba11ff-0f6a-4c50-ab34-6fbd286a643e"))
+			// 	}
+			// }
+		})
```

Comment:
Hmmm, looks unfinished? I guess here you don't care about log lines, just the error type returned. I assume you want to delete the commented-out lines and change the description in the It block.